src/Microsoft.IdentityModel.Protocols.OpenIdConnect/Configuration/OpenIdConnectConfigurationValidator.cs

```diff
@@ -39,11 +39,20 @@ public ConfigurationValidationResult Validate(OpenIdConnectConfiguration openIdC
                     Succeeded = false
                 };
             }
-            var numberOfValidKeys = openIdConnectConfiguration.JsonWebKeySet.Keys.Where(key => key.ConvertedSecurityKey != null).Count();
+
+            int numberOfValidKeys = 0;
+            for( int i = 0;  i < openIdConnectConfiguration.JsonWebKeySet.Keys.Count; i++)
```

Comment:
nit: spacing? `for( ` as opposed to `for ( `?

Reply:
+1 in favor of `for (`